analyze.module

```diff
@@ -25,4 +29,232 @@ function analyze_theme() {
       ],
     ],
   ];
-}
\ No newline at end of file
+}
+
+/**
+ * Implements hook_menu_local_actions_alter().
+ */
+function analyze_menu_local_actions_alter(&$local_actions): void {
```

Comment:
This _could_ be replaced by using a deriver where you'd include all the common elements in the yaml and a `deriver` key which reference a `DeriverInterface` to expand out things like params etc. See `field_ui` or `menu_test`.

Reply:
I'll todo it for future work.